src/assets/i18n/de.json5

```diff
@@ -731,9 +731,6 @@
   // "admin.workflow.title": "Administer Workflow",
   "admin.workflow.title": "Workflows verwalten",
 
-  // "admin.workflow.item.workflow": "Workflow",
-  "admin.workflow.item.workflow": "Workflows",
```

Comment:
@saschaszott : I'm curious why you removed this translation key, when it's obviously used.   Shouldn't we just change it's value to be "Workflow"?   That way it is clear that this key has been translated into German.

That said, I do agree that removing it should cause the UI to fallback to using English.  The behavior here is the same, but I'd prefer to keep the key since it is used.  That way if someone needs to modify the German translation of this key (for their own local purposes), they can find it in the `de.json5` file.

Reply:
Hi @tdonohue , I've removed the translation key as other (non-translated) keys, e.g. `admin.workflow.item.workspace`, are not included in `de.json5`.

Reply:
@tdonohue , should I better add **all** `admin.translation.*` translation keys in `de.json5` even if they are not translated?

Reply:
@saschaszott : Ideally, a key should be added to `de.json5` *minimally when it is translated*.  Optionally, they could be added in untranslated as well (to make things easier on future translators).

My confusion here is that I wasn't sure if the German translation of `admin.workflow.item.workflow` should also be `Workflow` (just like English).  If that is a correct German translation, then I'd recommend adding this key back into the `de.json5` file as:
```
"admin.workflow.item.workflow": "Workflow",
```

If that is NOT the correct translation, then your solution is OK for now.  We can remove it until it is translated.

Reply:
@tdonohue , "Workflows" is not the correct German translation of "Workflow". Currently, we have no better German translation of "Workflow" as well as "Workspace". 

The crucial point in this PR is that the current German translation of `admin.workflow.item.workflow` contains a plural form (trailing "s"). 

Consequently, as long as we do not have German translations of "Workflow" (and "Workspace") we should remove `admin.workflow.item.workflow` in `de.json5`.

Reply:
That sounds fine.  It's essentially one line change, so I can just accept it.  Sounds like I was confused about the intention here.